cmd/amocli/cmd/query/parcel.go

```diff
@@ -0,0 +1,35 @@
+package query
+
+import (
+	"encoding/hex"
+	"fmt"
+	"strings"
+
+	"github.com/spf13/cobra"
+
+	"github.com/amolabs/amoabci/client/rpc"
+)
+
+var ParcelCmd = &cobra.Command{
+	Use:   "parcel <parcelID>",
+	Short: "Get parcel's extra informations",
+	Args:  cobra.MinimumNArgs(1),
+	RunE:  parcelFunc,
+}
+
+func parcelFunc(cmd *cobra.Command, args []string) error {
+	tmp := strings.TrimLeft(args[0], "0x")
```

Comment:
What is this for?

Reply:
I thought parcelID might be formatted in hex string.

Reply:
This is the only place where `0x` prefix is used. For consistency, remove this prefix constraint.